### ISSUE TITLE
Feature Request: Saving and loading should be automatic

### ISSUE THREAD

**Issue by cary-rowen** [thgcode/snippetsForNVDA#27]
For memorized data, I don’t think we need to manually store it. This should be completely automatic, automatically saving when NVDA exits and automatically loading when NVDA starts.

1. The current reliance on manual data saving may cause the user to forget. NVDA will not ask the user when exiting like Notepad, resulting in data loss.
2. It takes up two extra gestures, which is not a big problem.
3. The new data will replace the old data. We only save the data of 10 slots and it is not expected to take up too much disk space.
4. We should consider situations where users try to save large amounts of text data, which may cause some performance issues:
    - NVDA+Shift+Ctrl+Number should only intercept a small part of the content when trying to report the content in the slot, otherwise it will definitely cause performance problems.
- Possible performance problems in other read, write or storage procedures.

**Comment by thgcode** [comment]
Maybe we could have a setting on NVDA settings for autosaving... I see benefits on the two approaches, manual saving could help if you want to quickly go back to the saved snippets, erasing the current ones, however it could cause problems if the user forgets to save the current ones, however automatically saving could save unwanted snippets in this case...
